.travis.yml

```diff
@@ -0,0 +1,18 @@
+os:
+  - linux
+  - osx
+language: generic
+sudo: required
+dist: trusty
+osx_image: xcode7.3
+install:
+  - echo "DEVELOPMENT-SNAPSHOT-2016-02-25-a" > .swift-version
```

Comment:
You could just use environment for this, swiftenv will install from `SWIFT_VERSION` env if found.

```yaml
env:
  - SWIFT_VERSION=DEVELOPMENT-SNAPSHOT-2016-02-25-a
```